src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepository.java

```diff
@@ -89,4 +89,12 @@
     int getIncompleteEnvelopesCountBefore(@Param("datetime") LocalDateTime dateTime);
 
     List<Envelope> findByContainerAndStatusAndZipDeleted(String container, Status status, boolean zipDeleted);
+
+    @Query("select e from Envelope e\n"
+        + "where e.zipFileCreateddate between :fromDateTime AND :toDateTime"
+    )
+    List<Envelope> getReceivedEnvelopes(
+        @Param("fromDateTime") Instant fromDateTime,
+        @Param("toDateTime") Instant toDateTime
```

Comment:
are those 2 parameters actually going to be used for something or can this method take LocalDate as parameter instead?